mk/kafka.mk

```
@@ -98,16 +22,16 @@ kafka-topics-list:  ## List the kafka topics from the kafka container
 
 .PHONY: kafka-topics-create
 kafka-topics-create:  ## Create the kafka topics in KAFKA_TOPICS
-	$(DOCKER) container inspect kafka-broker &> /dev/null || { echo "error:start kafka-broker container by 'make kafka-up'"; exit 1; }
+	$(DOCKER) container inspect $(KAFKA_CONTAINER_NAME) &> /dev/null || { echo "error:start kafka container by 'make kafka-up'"; exit 1; }
```

I suggest to reference `make compose-up`:

```suggestion
	$(DOCKER) container inspect $(KAFKA_CONTAINER_NAME) &> /dev/null || { echo "error:start kafka container by 'make compose-up'"; exit 1; }
```

---

I suggest to update `kafka-topics-list` rule to use `KAFKA_CONTAINER_NAME`.